src/main/markdown/using-intellij.md

```diff
@@ -0,0 +1,56 @@
+Using IntelliJ
+===
+
+GWT provides a set of tools that can simply be used with a
+text editor, the command line, and a browser. However, you may also use GWT with your
+favorite IDE. The Idea IntelliJ Ultimate Edition provides a GWT plugin, but the free Idea IntelliJ Community Edition 
+is all you need. IntelliJ provides an awesome Maven support. That is all, thatis  really necessary.
+
+*   [Download IntelliJ](#intellij)
+*   [Import a Web Application](#importing)
+*   [Run locally in Super Dev Mode](#running)
+
+## Download IntelliJ<a id="intellij"></a>
+
+If you do not already have IntelliJ, you may download it from the [IntgelliJ Website](https://www.jetbrains.com/idea/download/). 
+
+## Import a Web Application<a id="importing"></a>
+
+To import a Web Application - create with an archetype creator, select **File > New > Project from existing source** 
+from the File menu. Select the root directory of the project you want to import and press `enter`.
+
+A dialog opens:
+
+<div class="screenshot"><a href="images/import-into-intellij.png"><img src="images/import-into-intellij.png" alt="Screenshot" width="42%"/></a></div>
+
+Press `crate`. The import process starts and after a few seconds, the project window appears:
+
+<div class="screenshot"><a href="images/intellij-project.png"><img src="images/intellij-project.png" alt="Screenshot" width="42%"/></a></div>
+
+## Run locally in Super Dev Mode<a id="running"></a>
+
+To improve the development experience, you can set up to run configurations, one for the code server and another for the server.
+
+### Code Server Run Configuration
+
+Press `edit configuration`, a popup appears. Now press `+` and select `Maven`.  
+
+Enter `gwt:codeserver -pl mywebapp-client -am` in the field under run:
+
+<div class="screenshot"><a href="images/intellij-run-code-server.png"><img src="images/intellij-run-code-server.png" alt="Screenshot" width="42%"/></a></div>
+
+Now, you have a run configuration, that starts the code server.
+
+Note: you have to use the module name instead of '*'.
```

Comment:
I think you forgot to substitute the * above? It makes more sense below.

Reply:
added more text to clearify